Adapter/GroupedQuerySolariumAdapter.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace Markup\NeedleBundle\Adapter;
+
+use Pagerfanta\Adapter\AdapterInterface;
+use Pagerfanta\Exception\InvalidArgumentException;
+use Solarium\QueryType\Select\Query\Query;
+use Solarium\Core\Client\Client;
+use Solarium_Query_Select;
```

Comment:
Seems to be a lot of support for Solarium <3.0 here (the Zend style class names like `Solarium_Client`), which is unnecessary as this library officially only supports 3.0 and up just now (so, properly namespaced classes), with no intention of supporting anything lower.